biogeochem/EDLoggingMortalityMod.F90

```diff
@@ -510,17 +516,23 @@ subroutine logging_litter_fluxes(currentSite, currentPatch, newPatch, patch_site
       ! mass sent to that patch, by the area it will have remaining
       ! after it donates area.
       ! i.e. subtract the area it is donating.
-      
       remainder_area = currentPatch%area - patch_site_areadis
 
-
       ! Calculate the fraction of litter to be retained versus donated
       ! vis-a-vis the new and donor patch
-      
       retain_frac = (1.0_r8-harvest_litter_localization) * &
             remainder_area/(newPatch%area+remainder_area)
       donate_frac = 1.0_r8-retain_frac
 
+      if(remainder_area > rsnbl_math_prec) then
```

Comment:
we might want to have these types of checks in the fire and treefall mortality patch disturbance pipelines too

Reply:
I did a check, and we do use this logic for the fire and treefall routines.